docs/references/express/available-methods.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Express.js Available Methods
+description: Learn how to use the Clerk Express.js SDK to interact with the Clerk API.
+---
+
+# Express.js Available Methods
+
+All resource operations are mounted as sub-APIs on the `clerkClient` object. You can find the full list of available operations in the [JavaScript Backend SDK](/docs/references/backend/overview) documentation. To access a resource, you must first instantiate a `clerkClient` instance.
+
+## Instantiate a default `clerkClient` instance
+
+If you would like to use the default instance of `clerkClient` provided by the SDK, you can provide the `CLERK_SECRET_KEY` as an environment variable and instantiate `clerkClient` without passing configuration options.
+
+<CodeBlockTabs options={["ESM", "CJS"]}>
+
+  ```js
+  import { clerkClient } from '@clerk/express';
+
+  const userList = await clerkClient.users.getUserList();
+  ```
+
+  ```js
+  const pkg = require('@clerk/express');
+  const clerkClient = pkg.default;
+
+  clerkClient.sessions
+    .getSessionList()
+    .then((sessions) => console.log(sessions))
+    .catch((error) => console.error(error));
+  ```
+
+</CodeBlockTabs>
+
+## Instantiate a custom `clerkClient` instance
+
+If you would like to customize the behavior of the SDK, you can instantiate a `clerkClient` instance yourself by calling `createClerkClient` and passing in [options](#customizing-resources).
+
+The example below shows how to use `createClerkClient` to create a `clerkClient` instance and pass a Clerk secret key *instead of* setting a `CLERK_SECRET_KEY` environment variable.
+
+<InjectKeys>
+
+<CodeBlockTabs options={["ESM", "CJS"]}>
+```js
+import { createClerkClient } from '@clerk/express';
+
+const clerkClient = createClerkClient({ secretKey: '{{secret}}' });
+
+const clientList = await clerkClient.clients.getClientList();
+```
+
+```js
+const { createClerkClient } = require('@clerk/express/cjs');
```

Comment:
🔧 I tested it and we can use the following instead:
```js
const { createClerkClient } = require('@clerk/express');
```
Let's replace all references to the above